src/main.js

```diff
@@ -160,10 +161,10 @@ function showSavedPosters() {
   hide(mainPage);
   show(savedPage);
   savedGrid.innerHTML = ``
-  for ( var i = 0; i < savedPosters.length; i++){
+  for ( var i = 0; i < savedPosters.length; i++) {
     savedGrid.innerHTML += `
     <article class="mini-poster">
-      <img class="" src="${savedPosters[i].imageURL}" alt="nothin' to see here">
+      <img class="" id=${savedPosters[i].id} src="${savedPosters[i].imageURL}" alt="nothin' to see here">
```

Comment:
I would extract the id and src into variables and put them on line 161.5 like so `var posterId = savedPosters[i].id` then on line 167 you can say `id=${posterId}` and it will be just a little cleaner to read. Also some slightly more descriptive alt text would be good, even though you can't be super descriptive and reference every poster, maybe "Romance novel cover"?